.github/workflows/docs.yml

```
@@ -25,10 +25,16 @@ jobs:
         with:
-          python-version: "3.11"
-          cache: 'pip'
-          cache-dependency-path: |
+          python-version: "3.14"
+
+      - name: Set up uv
+        uses: astral-sh/setup-uv@v7
+        with:
+          cache-dependency-glob: |
             docs/requirements.txt
+          cache-suffix: ${{ matrix.python-version }}
```

actionlint (1.7.9)

property "python-version" is not defined in object type {}

(expression)